### PR TITLE
Update of user_auth_ldap.adoc

### PR DESCRIPTION
**Pull request by mmattel**
Improvements to `user_auth_ldap.adoc`

Backport to 10.5 and 10.6 necessary

### CODE REVIEW

**Comment by mmattel** [comment]
I have made an additional push (successfully) but it is not showing up here?

File: modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc [hunk 1]
```diff
@@ -65,15 +67,22 @@ IMPORTANT: The base DN and port are mandatory.
 
 ==== LDAPS Configuration
 
-LDAPS encrypts the connection between your LDAP server and ownCloud.
+LDAPS encrypts the connection between your LDAP server and ownCloud via SSL/TLS.
+service-ldap-with-tls-url
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc:71]: Can you really just mention this string without any sort of brackets around it?

Comment (mmattel) [inline modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc:71]:
```suggestion

```
no, many thinks for finding it, this was a left-over 🤦‍♂️

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@mmattel can you make the 2 backports? and I will review and merge.

**Comment by phil-davis** [comment]
Backports:
10.5 #3130 
10.6 #3131

**Pull request merged**